### PR TITLE
CNCF conformance results for k8s v1.27/ntnx-karbon

### PR DESCRIPTION
**Pull request by legacyrj**
Signed-off-by: legacyrj raj.surve@nutanix.com

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [ ] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [ ] Did you include the product/project logo in SVG, EPS or AI format?  
* [ ] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [ ] If your product/project is open source, did you include the `repo_url`?
* [ ] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

For a full list of requirements, please refer to these sections of the docs: [_Contents of the PR_](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#contents-of-the-pr), and [_Requirements_](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#requirements).

### CODE REVIEW

**Comment by taylorwaggoner** [comment]
@legacyrj this all looks great, but please fix the DCO check. Then i'll be able to merge. thanks!

**Comment by legacyrj** [comment]
@taylorwaggoner We have fixed the DCO issue. Merging is waiting on approval. Thank you.

**Comment by legacyrj** [comment]
Hello @Taylor ***@***.***>,

We have fixed the DCO check issue. Please confirm the status. It appears to be waiting on approval.

Appreciate your response.

Thank you.

Regards,
Raj Surve

From: Taylor Waggoner ***@***.***>
Date: Wednesday, January 10, 2024 at 3:40 PM
To: cncf/k8s-conformance ***@***.***>
Cc: Raj Surve ***@***.***>, Mention ***@***.***>
Subject: Re: [cncf/k8s-conformance] CNCF conformance results for k8s v1.27/ntnx-karbon (PR #2983)

@legacyrj [github.com]<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_legacyrj&d=DwMCaQ&c=s883GpUCOChKOHiocYtGcg&r=idbud97u5sXIs7LlTV2JfDck0T7ZtaaVQA7adjySsNc&m=VFYbKqMcoCrYG7dfzZ3n9OYKyVn5TCQnijUNV2Vyi3kxcaLb0Nk_unhP3m2XFiUi&s=pfGLhK6jNWdwGWwAqJ86VR8DzFVtUoff5QiicvDLuQk&e=> this all looks great, but please fix the DCO check. Then i'll be able to merge. thanks!

**Review by taylorwaggoner — APPROVED** [review]

**Pull request merged**
